src/pages/settings/ProfilePage.js

```diff
@@ -1,24 +1,236 @@
+                            </Text>
+                            <View style={[styles.flexRow, styles.mb6]}>
+                                <View style={styles.flex1}>
+                                    <Text style={[styles.mb1, styles.formLabel]}>First Name</Text>
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.firstName}
+                                        onChangeText={firstName => this.setState({firstName})}
+                                        placeholder="John"
+                                    />
+                                </View>
+                                <View style={[styles.flex1, styles.ml2]}>
+                                    <Text style={[styles.mb1, styles.formLabel]}>Last Name</Text>
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.lastName}
+                                        onChangeText={lastName => this.setState({lastName})}
+                                        placeholder="Doe"
+                                    />
+                                </View>
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Preferred Pronouns</Text>
+                                <View style={styles.mb1}>
+                                    <RNPickerSelect
+                                        onValueChange={pronouns => this.setState({pronouns, selfSelectedPronouns: ''})}
+                                        items={this.pronounDropdownValues}
+                                        style={styles.picker}
+                                        useNativeAndroidPickerStyle={false}
+                                        placeholder={{
+                                            value: '',
+                                            label: 'Select your pronouns',
+                                        }}
+                                        value={this.state.pronouns}
+                                        Icon={() => <Icon src={DownArrow} />}
+                                    />
+                                </View>
+                                {this.state.pronouns === pronounsMap.selfSelect ? (
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.selfSelectedPronouns}
+                                        onChangeText={selfSelectedPronouns => this.setState({selfSelectedPronouns})}
+                                        placeholder="Self-select your pronoun"
+                                    />
+                                ) : null}
```

Comment:
Same simplification here.
```suggestion
                                {this.state.pronouns === pronounsMap.selfSelect && (
                                    <TextInput
                                        style={styles.textInput}
                                        value={this.state.selfSelectedPronouns}
                                        onChangeText={selfSelectedPronouns => this.setState({selfSelectedPronouns})}
                                        placeholder="Self-select your pronoun"
                                    />
                                )}
```